tests/test_unit.py

```diff
@@ -4,18 +4,20 @@
 
 
 Sample = namedtuple('Sample', ['strain', 'medium', 'feed_medium', 'read_scalars', 'name'])
-Strain = namedtuple('Strain', ['organism', 'parent_strain', 'genotype'])
+Strain = namedtuple('Strain', ['organism', 'parent_strain', 'pool', 'parent_pool', 'genotype'])
 Medium = namedtuple('Medium', ['read_contents'])
 Product = namedtuple('Product', ['chebi_id', 'chebi_name'])
 Phase = namedtuple('Phase', ['id', 'title', 'start', 'end'])
 Organism = namedtuple('Organism', ['short_code'])
+Pool = namedtuple('Pool', ['parent_pool', 'genotype'])
 
 organism = Organism('ECO')
 p1 = Product(16828, 'first')
 p2 = Product(17895, 'second')
 medium = Medium(lambda: [{'compound': p1, 'concentration': 0.01}, {'compound': p2, 'concentration': 0.02}])
 phase = Phase(1, 'phase1', 0, 14)
-strain = Strain(organism, None, '+Aac')
+pool = Pool(None, '')
+strain = Strain(organism, None, pool, None, '+Aac')
```

Comment:
would be perfect to add a test case on making sure that overall genotype changes have both pool and strain changes, like
```
pool = Pool(None, '+poolgene')
strain = Strain(organism, None, pool, None, '+Aac')
assert(extract_genotype_changes(strain) == [ '+poolgene',  '+Aac'])
```